deploy/template.yaml

```diff
@@ -101,8 +101,9 @@ Mappings:
       DNSSUFFIX: review-o.dev.account.gov.uk
       MESSAGERETENTIONPERIODDAYS: 345600 # Default: 4 days
       TXMAMESSAGERETENTIONPERIODDAYS: 604800 # Default: 7 days
-      GOVUKNOTIFYAPI: "https://f2f-gov-notify-stub-govnotifystub.review-o.dev.account.gov.uk/govnotify"
-      GOVUKNOTIFYTEMPLATEID: "bea2a584-4dca-4970-a90d-93977e752fdf"
+      GOVUKNOTIFYAPI: "https://api.notifications.service.gov.uk"
+      GOVUKNOTIFYPDFTEMPLATEID: "bea2a584-4dca-4970-a90d-93977e752fdf"
+      GOVUKNOTIFYREMINDERTEMPLATEID: "1490de9b-d986-4404-b260-ece7f1837115"
```

Comment:
I assume there is but just want to triple check, there is a reason we've only added this in dev, yes?

Reply:
Added for all lower envs, pending Prod

Reply:
ty - is there a ticket to add the prod one or is that going to hold up this PR?

Reply:
It'll hold up this PR, this lambda runs on a cron so once deployed it'll start running immediately at 10am each weekday

Reply:
Template IDs updated now